src/Gui/ExpressionCompleter.cpp

```diff
@@ -280,6 +283,10 @@ ExpressionLineEdit::ExpressionLineEdit(QWidget *parent)
     , block(false)
 {
     connect(this, SIGNAL(textChanged(const QString&)), this, SLOT(slotTextChanged(const QString&)));
+
+    //enable partial completion and set the required regex by default
+    enablePartialCompletion();
+    setPartFilterSettings(QRegExp(tr("[+-*=!<>^?: ()/]")), QString::SkipEmptyParts);
```

Comment:
You must not use tr() here because this marks text (i.e. all the operator symbols) for translations. Instead use QString::fromLatin1()